qa/L0_input_validation/test.sh

```diff
@@ -80,49 +81,6 @@ wait $SERVER_PID
 pip install torch
 pip install pytest-asyncio
 
-mkdir -p models/pt_identity/1
-PYTHON_CODE=$(cat <<END
-import torch
-torch.jit.save(
-    torch.jit.script(torch.nn.Identity()),
-    "`pwd`/models/pt_identity/1/model.pt",
-)
-END
-)
-res="$(python3 -c "$PYTHON_CODE")"
-
-if [ $? -ne 0 ]; then
-    echo -e "\n***\n*** model "pt_identity" initialization FAILED. \n***"
-    echo $res
-    exit 1
-fi
-
-# Create the config.pbtxt file with the specified configuration
-cat > models/pt_identity/config.pbtxt << EOL
-name: "pt_identity"
-backend: "pytorch"
-max_batch_size: 8
-input [
-  {
-    name: "INPUT0"
-    data_type: TYPE_FP32
-    dims: [8]
-  }
-]
-output [
-  {
-    name: "OUTPUT0"
-    data_type: TYPE_FP32
-    dims: [8]
-  }
-]
-# ensure we batch requests together
-dynamic_batching {
-    max_queue_delay_microseconds: 1000000
-}
-EOL
-
-cp -r $DATADIR/qa_model_repository/graphdef_object_int32_int32 models/.
```

Comment:
Why removed?

Reply:
Use model "simple_identity" instead to test string inputs.